### ISSUE TITLE
Update to binocle 0.2.0 via cargo-install fails

### ISSUE THREAD

**Issue by virtualritz** [sharkdp/binocle#43]
```
❯ cargo install-update --all
    Updating registry 'https://github.com/rust-lang/crates.io-index'

Package                Installed  Latest   Needs update
binocle                v0.1.2     v0.2.0   Yes
[...]

Updating binocle
    Updating crates.io index
  Installing binocle v0.2.0
  [...]
   Compiling binocle v0.2.0
error[E0432]: unresolved imports `clap::crate_description`, `clap::crate_version`
 --> /home/mark/.cargo/registry/src/github.com-1ecc6299db9ec823/binocle-0.2.0/src/options.rs:1:12
  |
1 | use clap::{crate_description, crate_version, Parser};
  |            ^^^^^^^^^^^^^^^^^  ^^^^^^^^^^^^^ no `crate_version` in the root
  |            |
  |            no `crate_description` in the root

error: cannot find derive macro `Parser` in this scope
 --> /home/mark/.cargo/registry/src/github.com-1ecc6299db9ec823/binocle-0.2.0/src/options.rs:4:10
  |
4 | #[derive(Parser)]
  |          ^^^^^^
  |
note: `Parser` is imported here, but it is only a trait, without a derive macro
 --> /home/mark/.cargo/registry/src/github.com-1ecc6299db9ec823/binocle-0.2.0/src/options.rs:1:46
  |
1 | use clap::{crate_description, crate_version, Parser};
  |                                              ^^^^^^

error: cannot find attribute `clap` in this scope
 --> /home/mark/.cargo/registry/src/github.com-1ecc6299db9ec823/binocle-0.2.0/src/options.rs:5:3
  |
5 | #[clap(version = crate_version!(), about = crate_description!())]
  |   ^^^^
  |
  = note: `clap` is in scope, but it is a crate, not an attribute

error: cannot find attribute `clap` in this scope
 --> /home/mark/.cargo/registry/src/github.com-1ecc6299db9ec823/binocle-0.2.0/src/options.rs:8:7
  |
8 |     #[clap(
  |       ^^^^
  |
  = note: `clap` is in scope, but it is a crate, not an attribute

error[E0599]: no function or associated item named `parse` found for struct `CliOptions` in the current scope
  --> /home/mark/.cargo/registry/src/github.com-1ecc6299db9ec823/binocle-0.2.0/src/main.rs:15:31
   |
15 |     let options = CliOptions::parse();
   |                               ^^^^^ function or associated item not found in `CliOptions`
   |
  ::: /home/mark/.cargo/registry/src/github.com-1ecc6299db9ec823/binocle-0.2.0/src/options.rs:6:1
   |
6  | pub struct CliOptions {
   | --------------------- function or associated item `parse` not found for this
   |
   = help: items from traits can only be used if the trait is implemented and in scope
   = note: the following traits define an item `parse`, perhaps you need to implement one of them:
           candidate #1: `Parser`
           candidate #2: `nom::internal::Parser`
           candidate #3: `nom::sequence::Tuple`
           candidate #4: `ttf_parser::parser::FromData`
           candidate #5: `ttf_parser::parser::FromData`
           candidate #6: `ttf_parser::parser::FromSlice`
           candidate #7: `ttf_parser::ggg::lookup::LookupSubtable`
           candidate #8: `ttf_parser::ggg::layout_table::RecordListItem`
 ```

**Comment by sharkdp** [comment]
Hm. Looks like you are trying to build the source code of 0.2.0 with dependencies from master. What is `cargo install-update`? Does it have a `--locked` option to use the exact dependencies from the tagged commit?

**Comment by virtualritz** [comment]
The standard way to install `binocle` as a binary published on `crates.io` is `cargo install binocle`.

And if you want to update everything you installed via `cargo-install` thusly, you use `cargo install-update --all`. It runs via `cron` on all my machines once a day to check.

**Comment by virtualritz** [comment]
I forgot: no `--locked` feature on `cargo-update`.

The build breaks as above when doing
```
cargo install binocle
```

Workaround to update an existing installation:
```
cargo uninstall binocle
cargo install --locked binocle
```

While this works it is extremely uncommon to run `cargo-install` with any parameters when using it to install binaries from `crates.io`. I.e. from my experience, stuff like `--locked` is commonly used when using `cargo-install` for a binary target on a local repo.

Which I guess would be the reason `cargo-update` doesn't support the `--locked` flag.

I.e. I would still suggest to fix `0.2.0` to install w/o `--locked`.

**Comment by sharkdp** [comment]
Well. `--locked` is normally not needed, unless one of our dependencies has a semver violation (at least that is my interpretation). We therfore suggest to use `--locked` when installing `bat`, for example.

But I agree this should be fixed (maybe it is already fixed on master by the update to `clap` 3). Maybe it needs an upgrade of another dependency. Any help would be appreciated.

**Comment by sharkdp** [comment]
Should be fixed in v0.3.0
